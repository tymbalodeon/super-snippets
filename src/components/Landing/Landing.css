.Landing > h2 {
  text-align: center;
}
