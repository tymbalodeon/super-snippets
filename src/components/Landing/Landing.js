import React, { Component } from 'react';

export default class NotFoundPage extends Component {
  render() {
    return (
      <section className="Landing">
        <h2>Welcome to SuperSnippets</h2>
        <p>Please Login or Register above.</p>
        <p>This is a description of the site.</p>
      </section>
    );
  }
}
