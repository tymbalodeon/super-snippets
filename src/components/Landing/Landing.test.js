import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Landing from './Landing';

describe('<Landing />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');

    ReactDOM.render(<Landing />, div);

    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer.create(<Landing />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
