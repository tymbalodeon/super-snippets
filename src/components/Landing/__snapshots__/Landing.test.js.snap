// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing /> renders the UI as expected 1`] = `
<section
  className="Landing"
>
  <h2>
    Welcome to SuperSnippets
  </h2>
  <p
    className="please-login"
  >
    Please Login or Register above.
  </p>
  <p
    className="landing-p"
  >
    This is a place to sketch, store, organize, and share musical ideas. Currently, users can create snippets, edit them, delete them, tag them by project and view them (either all or by project). The goal is that snippets could eventually be run in the browser, perhaps using
     
    <a
      href="https://supercollider.github.io/"
      rel="noopener noreferrer"
      target="_blank"
    >
      SuperCollider
    </a>
     
    to generate sound or
     
    <a
      href="http://lilypond.org/"
      rel="noopener noreferrer"
      target="_blank"
    >
      LilyPond
    </a>
     
    to generate music notation in pdf form.
  </p>
  <div
    className="test"
  >
    <p
      className="test-text"
    >
      Test out the site by logging in with:
    </p>
    <p
      className="demo"
    >
      User name: demo
    </p>
    <p
      className="demo"
    >
      Password: password
    </p>
  </div>
</section>
`;
